src/main/kotlin/net/maizegenetics/phgv2/cli/CreateFastaFromHvcf.kt

```diff
@@ -67,37 +67,56 @@ class CreateFastaFromHvcf : CliktCommand( help = "Create a fasta file from a hvc
     val hvcfFile by option("--hvcf-file", help = "hVCF file to import instead of hitting TileDB")
         .default("")
 
+    val hvcfDir by option("--hvcf-dir", help = "hVCF directory to import instead of hitting TileDB")
+        .default("")
```

Comment:
I think this would be clearer if it said something like "Path to directory holding hVCF files.  Data wlll be pulled directly from these files instead of querying TileDB"

Similar change to --hvcf-file,  reading something like "Path to hVCF file to import.  Data will be pulled from this file instead of querying TileDB"